backend/src/main/java/ai/verta/modeldb/App.java

```diff
@@ -420,7 +426,11 @@ private static void initializeRelationalDBServices(
             authService, roleService, repositoryDAO, commitDAO, blobDAO, metadataDAO);
     ProjectDAO projectDAO =
         new ProjectDAORdbImpl(authService, roleService, experimentDAO, experimentRunDAO);
-    ArtifactStoreDAO artifactStoreDAO = new ArtifactStoreDAORdbImpl(artifactStoreService);
+    ArtifactStoreDAO artifactStoreDAO = new ArtifactStoreDAODisabled();
```

Comment:
initialize it to null and have a clear if else

Reply:
Fixed